### PR TITLE
Use latest k8s for operator example to work

### PR DESCRIPTION
**Pull request by moberghammer**
## Description
Use latest k8s version in example for it to work.

## Related Issue(s)

- Fixes https://github.com/vitessio/vitess/issues/13094 

## Checklist

-   [x] "Backport to:" labels have been added if this change should be back-ported
-   [x] Tests were added or are not required
-   [x] Did the new or modified tests pass consistently locally and on the CI
-   [x] Documentation was added or is not required

### CODE REVIEW

File: examples/operator/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ```
 # Start minikube
-minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.19.5
+minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.26.3
```
Comment (frouioui) [inline examples/operator/README.md:5]: The latest supported version of the operator is `1.25.8` (https://github.com/planetscale/vitess-operator#compatibility), let's change it to that.

```suggestion
minikube start --cpus=8 --memory=11000 --disk-size=50g --kubernetes-version=v1.25.8
```

**Review by frouioui — APPROVED** [review]

**Comment by frouioui** [comment]
@moberghammer looks good to me, thank you for your contribution. Can you please also open an issue to describe the problem? We'll remove the `NeedsIssue` label once that is done.

**Comment by frouioui** [comment]
While we are at it, do you mind modifying the `Kubernetes Getting Started Guide` page in our documentation? (https://vitess.io/docs/16.0/get-started/operator/), it still shows k8s `v1.24`. We would need to change it on both `v16.0` and `v17.0` docs.

**Comment by moberghammer** [comment]
> While we are at it, do you mind modifying the `Kubernetes Getting Started Guide` page in our documentation? (https://vitess.io/docs/16.0/get-started/operator/), it still shows k8s `v1.24`. We would need to change it on both `v16.0` and `v17.0` docs.

Where do I modify these guides?

**Comment by frouioui** [comment]
> Where do I modify these guides?

That would be in https://github.com/vitessio/website/blob/prod/content/en/docs/16.0/get-started/operator.md and https://github.com/vitessio/website/blob/prod/content/en/docs/17.0/get-started/operator.md

**Comment by frouioui** [comment]
Thanks for the issue @moberghammer!

**Comment by moberghammer** [comment]
> > Where do I modify these guides?
> 
> That would be in https://github.com/vitessio/website/blob/prod/content/en/docs/16.0/get-started/operator.md and https://github.com/vitessio/website/blob/prod/content/en/docs/17.0/get-started/operator.md

https://github.com/vitessio/website/pull/1472

**Comment by frouioui** [comment]
Thank you @moberghammer, website Pull Request is merged.

**Review by mattlord — APPROVED** [review]
Thanks @moberghammer ! ❤️

**Pull request merged**